xpdtools/pipelines/qoi.py

```diff
@@ -29,3 +30,44 @@
 
 pdf_argrelmax_kwargs = pdf_peaks.upstreams[0].kwargs
 mean_argrelmax_kwargs = mean_peaks.upstreams[0].kwargs
+
+
+def max_value(pdf):
```

Comment:
these need docstrings.

Reply:
I believe this is the old commit. The more recent commit has these functions moved to the tools.py file with docstrings.